test/test.js

```diff
@@ -188,29 +190,59 @@ describe('cypress-terminal-report', () => {
   }).timeout(60000);
 
   it('Should generate proper log output files.', async () => {
-    const outRoot = __dirname + '/output';
+    const outRoot = path.join(__dirname, 'output');
     const testOutputs = ['out.txt', 'out.json', 'out.cst'];
     testOutputs.forEach((out) => {
-      if (fs.existsSync(outRoot + '/' + out)) {
-        fs.unlinkSync(outRoot + '/' + out);
+      if (fs.existsSync(path.join(outRoot, out))) {
+        fs.unlinkSync(path.join(outRoot, out));
       }
     });
 
-    if (fs.existsSync(outRoot + '/not')) {
-      fs.rmdirSync(outRoot + '/not', { recursive: true });
+    if (fs.existsSync(path.join(outRoot, 'not'))) {
+      fs.rmdirSync(path.join(outRoot, 'not'), { recursive: true });
     }
 
     const clean = (str) =>
       // Clean error trace as it changes from test to test.
       str.replace(/at [^(]+ \([^)]+\)/g, '');
 
-    await runTest(commandBase(['generateOutput=1'], ['requests.spec.js', 'happyFlow.spec.js']), (error, stdout, stderr) => {
+    const osSpecificEol = (str) => 
+      // Change line endings to win32 if needed
+      (os.EOL === '\r\n' ? str.replace(/\n/g, '\r\n') : str);
+
+    const specFiles = ['requests.spec.js', 'happyFlow.spec.js'];
+    await runTest(commandBase(['generateOutput=1'], specFiles), (error, stdout, stderr) => {
       testOutputs.forEach((out) => {
-        const expected = fs.readFileSync(outRoot + '/' + out.replace(/\.([a-z]+)$/, '.spec.$1'));
-        const value = fs.readFileSync(outRoot + '/' + out);
+        const expectedBuffer = fs.readFileSync(
+          path.join(outRoot, out.replace(/\.([a-z]+)$/, '.spec.$1'))
+        );
+        const valueBuffer = fs.readFileSync(path.join(outRoot, out));
+        let value = clean(valueBuffer.toString());
+        if (path.sep === '\\') {
```

Comment:
I think it would be best if our outputs were not OS specific. It would also simplify the tests, and would simplify code relying on the output format of these files. But I am open to counter arguments.

Reply:
Sorry about the delay, I was busy with other tasks.

For text files, I think it makes most sense to follow the EOL conventions of the OS. If you open a text file in the default text reader in Windows (Notepad), it doesn't handle LF EOL so everything ends up as one very long line. Yes, you can probably expect developers to work around this with ease (VS Code and Visual Studio handle LF EOL fine), but the output of this plugin might just as well be used by QA, that in many cases don't use developer IDEs to read files.

A possible alternative could be to have a different outputTarget for CRLF EOL, such that txt always renders LF EOL, while 'crlfTxt' (or 'winTxt' or whatever makes sense) renders CRLF EOL. That way the user can pick what they need. The downside is that this makes the configuration OS specific which doesn't help for organizations where devs and QA may be running different OS.